### PR TITLE
Creating a stub of the file for burbank-ca.md

### LINKED ISSUE #42: Create a new file for githubteacher hometown
## Add file for Fuquay NC
- [x] Create a branch
- [x] Add my file
- [x] Create a commit
- [x] Open a pull request
- [x] Have a discussion about the pull request
- [x] Merge our pull request

### PR DESCRIPTION
**Pull request by lmalgeri**
Adding sub sections for what we want to know about Burbank-ca.md

This pull request closes #42

### CODE REVIEW

File: burbank-ca.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Burbank, CA
+
```
Comment (lmalgeri) [inline burbank-ca.md:2]: a new comment for practice

File: burbank-ca.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+### Great places to eat
```
Comment (alexladines) [inline burbank-ca.md:3]: I hang out in Burbank all the time as I went to high school in Panorama City! What's your favorite place to eat?

Comment (lmalgeri) [inline burbank-ca.md:3]: I don't know. What's your favorite place?

Comment (alexladines) [inline burbank-ca.md:3]: Probably Portos!

**Comment by alexladines** [comment]
This Looks good!

**Pull request merged**